src/services/product.service.ts

```
@@ -0,0 +1,33 @@
+import { HttpException, Injectable } from "@nestjs/common";
+import { prisma } from "src/db/prisma";
+import { ProductRequest, ProductType } from "src/db/product.schema";
+import { v4 as uuidv4} from "uuid";
+
+@Injectable()
+export class ProductService {
+    async createProduct(params: ProductRequest): Promise<ProductType> {
```

ESLint

[error] Delete `··`

(prettier/prettier)